src/args.rs

```diff
@@ -111,134 +137,91 @@ pub struct CreateCommand {
     pub overwrite: bool,
 
     /// Allow installation on non-removable devices. Use with extreme caution!
-    ///
-    /// If no device is specified in the command line, the device selection menu will
-    /// show non-removable devices
     #[clap(long = "allow-non-removable")]
     pub allow_non_removable: bool,
 
     /// The AUR helper to install for handling AUR packages.
-    #[clap(
-        value_enum,
-        long = "aur-helper",
-        default_value_t = AurHelper::Paru,
-        ignore_case = true
-    )]
+    #[clap(long = "aur-helper", value_enum, default_value_t = AurHelper::Paru, ignore_case = true)]
```

Comment:
should be `yay` by default in amarchi, I think

Reply:
Thanks, this helps simplify it a bit too.